internal/courses/courses.go

```diff
@@ -1,22 +1,22 @@
-package Courses
+package courses
 
 import "time"
 
-type CourseId string
+type CourseName string
 type CourseSubject string
-type CourseCatalog int
```

Comment:
`CourseCatalog` is like the `136` in `MATH 136`, pretty sure we still need it. Thanks for adding `CourseName` lol, almost forgot

Reply:
fixed